cookbooks/aws-parallelcluster-platform/resources/nvidia_driver/partial/_nvidia_driver_common.rb

```diff
@@ -50,17 +50,26 @@
   end
 
   if set_compiler?
-    package "gcc10" do
+    package compiler_version do
       retries 10
       retry_delay 5
     end
-    cookbook_file 'dkms/nvidia.conf' do
-      source 'dkms/nvidia.conf'
+    # TODO: Move to install_packages_alinux2023
```

Comment:
why not moved?

Reply:
This comment is no longer relevant, during development I added this as I thought we should install these anyway.
However, As these packages are required only in case of we install NVIDIA, I dont see the relevance of adding them by default.

Reply:
Removed the commet